docs/streams/upgrade-guide.html

```diff
@@ -93,6 +95,12 @@ <h1>Upgrade Guide and API Changes</h1>
     </p>
 
     <h3><a id="streams_api_changes_300" href="#streams_api_changes_300">Streams API changes in 3.0.0</a></h3>
+    <p>
+      The <code>StreamsConfig.EXACTLY_ONCE</code> and <code>StreamsConfig.EXACTLY_ONCE_BETA</code> configs have been deprecated, and a new <code>StreamsConfig.EXACTLY_ONCE_V2</code> config has been
```

Comment:
I feel like it does make more sense to include the actual variable here, since that's what's being deprecated. Also that way no one has to pause and think "hm, is that in StreamsConfig or ConsumerConfig or..."

Reply:
Well, we do deprecate `StreamsConfig.EXACTLY_ONCE`, too, but user might just do `config.put("processing.guarantee", "exactly_once");` (or have a config file with `"exactly_once"`) in it. To me, the main change is that the config itself is deprecated and the deprecation of variable is just a "side effect".

Reply:
I guess I'll just try to mention StreamsConfig in there somewhere...